src/components/Note/Note.tsx

```diff
@@ -7,29 +7,37 @@ export interface INoteProps {
   type: 'primary' | 'positive' | 'negative' | 'warning';
   error: Error;
   resetErrorBoundary: () => void;
+  dismissable: boolean;
 }
 
 export function IxNote({
   error,
   type,
   resetErrorBoundary,
+  dismissable,
 }: INoteProps): ReactElement {
-  const [message, _link, ...rest] = error.message.split('$');
+  const [message, _link, mid, _secondLink, ...rest] = error.message.split('$');
```

Comment:
Doubling checking; this change doesn't break parsing errors with only one URL correct? From what I can tell it does not but hard to be sure just from reading.

Reply:
Still works with one URL, yeah.

<img width="631" alt="Screen Shot 2022-11-16 at 12 41 13 PM" src="https://user-images.githubusercontent.com/8823474/202254462-be202540-67aa-4b9a-879a-0a0f34a64294.png">